packages/react-native-renderer/src/server/ReactNativeServerFormatConfig.js

```diff
@@ -204,33 +204,40 @@ export function writePlaceholder(
 export function writeStartCompletedSuspenseBoundary(
   destination: Destination,
   responseState: ResponseState,
-  id: SuspenseBoundaryID,
 ): boolean {
-  writeChunk(destination, SUSPENSE_COMPLETE);
-  return writeChunk(destination, formatID(id));
+  return writeChunk(destination, SUSPENSE_COMPLETE);
 }
+
+export function pushStartCompletedSuspenseBoundary(
+  target: Array<Chunk | PrecomputedChunk>,
+): void {
+  target.push(SUSPENSE_COMPLETE);
+}
+
 export function writeStartPendingSuspenseBoundary(
   destination: Destination,
   responseState: ResponseState,
   id: SuspenseBoundaryID,
 ): boolean {
-  writeChunk(destination, SUSPENSE_PENDING);
-  return writeChunk(destination, formatID(id));
```

Comment:
This should still be using ID since we'll have to turn it from pending to one of the other statuses at some point and to do that we need to refer to it by ID.

Reply:
Oops. That was unintentional, though I did notice that ReactDOMServerFormatConfig doesn't include the ID in `writeStartPendingSuspenseBoundary`,

Reply:
Yea that’s because it uses the assignID mechanism that tries to optimize this by stashing the ID inside the fallback.

However that didn’t work out and is ripe for refactoring. I suspect what we’ll do is just write the ID in there instead.